### PR TITLE
Add python-raven 6.6.0

### PR DESCRIPTION
**Pull request by gongled**
Hello, @andyone.

In addition to SaltStack which has been already packed into RPM using Python 3.4 from EPEL, I suggest you to put Raven python library to EK repository. It provides Python bindings for Sentry application monitoring. Raven is useful for those people who collect metrics via [SaltStack returner](https://docs.saltstack.com/en/latest/ref/returners/all/salt.returners.sentry_return.html).

I am not sure how should we prepare Python packages for EL6/EL7. Should we provide packages with Python 2.x? Or is it enough to pack only Python 3.x? @andyone what do you think?

### CODE REVIEW

**Comment by andyone** [comment]
> Should we provide packages with Python 2.x?

Yes, we should if this package supports Python 2.6.6 (_the latest available version of Python 2.x on CentOS 6_) without any patches. This information is usually can be found in `setup.py` file.

File: python34-raven/python34-raven.spec [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+and Flask. Raven also includes drop-in support for any WSGI-compatible web
+application.
+
+################################################################################
+
+%prep
+%setup -q -n %{pkg_name}-%{version}
+%patch0 -p1
+
+%{__rm} -f %{short_name}/data/cacert.pem
```
Comment (andyone) [inline python34-raven/python34-raven.spec]: Use `rm` instead of macro `%{__rm}`

File: python34-raven/python34-raven.spec [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+URL:              https://pypi.python.org/pypi/raven/
+
+Source0:          https://github.com/getsentry/%{pkg_name}/archive/%{version}.tar.gz
+
+Patch0:           raven-use-system-cacert.patch
+
+BuildRoot:        %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+BuildArch:        noarch
+
+Requires:         python-contextlib2
```
Comment (andyone) [inline python34-raven/python34-raven.spec]: Maybe it should be `python34-contextlib2`?

**Review by andyone — CHANGES_REQUESTED** [review]

File: python34-raven/python34-raven.spec (hunk 2 above)
Comment (gongled) [reply python34-raven/python34-raven.spec]: I removed this dependency due to none usage. I found that `python-contextlib2` requires only with unit-tests for Python 2.x.

**Comment by andyone** [comment]
@gongled Could you please move out package for Python 3.4 to separate file? Currently, all our packages for Python 3.4 have their specs.

**Comment by gongled** [comment]
Sure. I did it (ef7434c43592cebea9cf34aed7c7a91eb814aefb).

**Review by andyone — APPROVED** [review]

**Comment by andyone** [comment]
Thanks. Looks good, merged.

**Pull request merged**